app/src/js/utils/table-config/reconciliation-reports.js

```diff
@@ -85,10 +85,24 @@ const handleCsvDownloadClick = (e, reportName, dispatch) => {
 const handleDeleteClick = (e, value, dispatch, query) => {
   e.preventDefault();
   dispatch(deleteReconciliationReport(value)).then((response) => {
-    dispatch(listReconciliationReports(query));
+    refreshReports(dispatch, value, query);
   });
 };
 
+// It can take an extra second or two to delete a report from
+// Elasticsearch. This function keeps checking the reports to make sure it was
+// deleted before displaying the delete result.
+const refreshReports = (dispatch, value, query) => {
+  dispatch(listReconciliationReports(query)).then((reports) => {
```

Comment:
This should probably have a hard limit to the number of retries and maybe a very small delay. 

My first testing attempt was to delete a badly named record, and this went into an infinite loop. 
Maybe use p-retry with small backoff or something like that? It seems like it's calling a bunch of times (3-4ish) on a successful delete, maybe start with a 250ms or 500ms delay? what do you think?

Reply:
You can recreate by creating a report with a name with slashes in it (there is a ticket to fix this).  e.g. "01/20/2021 report name"

Reply:
Ah I thought this would fail like normal. Bad assumption on my part. And yeah it takes 2-4 calls to finally get it right but let me try adding a delay to see if it works out better.

Reply:
I'm also little concerned that it does fail, but we don't seem to do anything about the failure.  But that's out of scope here.